client/deposits/overview/instant-deposit-button.js

```diff
@@ -29,34 +66,68 @@ const InstantDepositModal = ( { amount, fee, net, onClose, onSubmit, inProgress
 					currency.formatCurrency( net / 100 )
 				) }
 			</Button>
+			<Button isDefault onClick={ onClose }>
+				{ __( 'Cancel', 'woocommerce-payments' ) }
+			</Button>
 		</Modal>
-	)
-}
+	);
+};
+
+const InstantDepositSubmitNotice = ( { error } ) => {
+	let message = '';
+	// TODO: supply proper error notices.
+	switch ( error.code ) {
+		case 'balance_insufficient':
```

Comment:
This shouldn't happen at all, because the server checks the balance available for Instant Payout on the endpoint that's called for rendering the Deposits page. There's probably some error in the server logic that should be fixed. Let's remove this case from the client and just show a generic error.

Reply:
I went off of that specific error due to it's the one I was receiving. I removed the case and just left the default for now. 

Just to note that this relies on https://github.com/Automattic/woocommerce-payments/pull/1343 in order to get the proper error code, otherwise a generic error code is always returned.